queries/team.sql

```diff
@@ -1,40 +1,50 @@
-) issue
-GROUP BY id,name,parent_id, org_id
-UNION ALL 
-SELECT
-id,
-name,
-parent_id,
-org_id
-FROM
-(
-  SELECT
-      source,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.id') end as id,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.name') end as name,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.slug') end as slug,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.parent') end as parent_id,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.privacy') end as privacy,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.action') end as action,
-      CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.organization.id') end as org_id,
-      FROM `mgt-fourkeys-8d87d44d.four_keys.events_raw` WHERE event_type = "team"
-) issue where action in ('injected','created')
-GROUP BY
-id,
-name,
-parent_id,
-org_id
\ No newline at end of file
+    id,
+    name,
+    'NA' as parent_id,
+    'NA' as org_id,
+    'NA' as repository_id,
+    'NA' as slug,
+    'NA' as action,
+    max(time_created) as time_created
+    FROM
+    (
+        SELECT time_created,
+        CASE WHEN source LIKE "pagerduty%" THEN JSON_EXTRACT_SCALAR(metadata, '$.event.data.teams[0].id') end as id,
+        CASE WHEN source LIKE "pagerduty%" THEN JSON_EXTRACT_SCALAR(metadata, '$.event.data.teams[0].summary') end as name,
+        FROM `mgt-fourkeys-8d87d44d.four_keys.events_raw`
+        WHERE event_type like "incident%" 
+    ) issue
+    GROUP BY id,name,parent_id, org_id,repository_id, slug
+    UNION ALL 
+    SELECT
+    id,
+    name,
+    parent_id,
+    org_id,
+    repository_id,
+    slug,
+    action,
+    max(time_created) as time_created,
+    FROM
+    (
+    SELECT
+        source, time_created,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.id') end as id,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.name') end as name,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.slug') end as slug,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.parent') end as parent_id,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.team.privacy') end as privacy,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.action') end as action,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.organization.id') end as org_id,
+        CASE WHEN source LIKE "github%" THEN JSON_EXTRACT_SCALAR(metadata, '$.repository.id') end as repository_id,
+        FROM `mgt-fourkeys-8d87d44d.four_keys.events_raw` WHERE event_type = "team"
+    ) issue 
+    GROUP BY
+    id,
+    name,
+    parent_id,
+    org_id,
+    repository_id,
+    slug,
+    action
```

Comment:
Can you add the extra line to remove the github error please.

Reply:
my bad, sorry i have added extra line.